### PR TITLE
#10218_warning_message_not_displayed added a warning message when use…

### LINKED ISSUE #10218: The 'No event participants selected' warning message is not displayed when the user clicks on 'Create Contacts' but doesn't previously select any Event Participant
### Bug Description
When no event participant is selected in Bulk Edit mode, and the user clicks on Bulk Actions -> 'Create Contacts', the 'No event participants selected' warning message should be displayed, like for the other actions (like 'Delete' or 'Create quarantine order documents'), but is not.

### Steps to Reproduce

1. Open an Event and select the Event Participants tab;
2. Enable the Bulk Edit mode;
3. Do not select any Event Participant and click on 'Bulk Actions' and then on 'Create Contacts';

### Expected Behavior
The 'No event participants selected' warning message should be displayed;

### Expected Behavior
Nothing happens.

### Screenshots

create contacts for EP

### System Details

* Device: Windows 10
* SORMAS version: 1.74.3 and after
* Android version/Browser: Chrome
* Server URL:  test1402.sormas.netzlink.com
* User Role: Admin, National User

### Additional Information
Discovered when testing ticket #3845

### PR DESCRIPTION
**Pull request by valentinmikleuvg**
…r click on Bulk Actions -> Create Contacts with no event participant selected

Fixes #10218

### CODE REVIEW

**Review by StefanKock — COMMENTED** [review]
The commit message does not contain the ticket id.

Rewrite the commit message to:
```
#10218 added a warning message when user click on Bulk Actions

-> Create Contacts with no event participant selected`
```

**Review by StefanKock — CHANGES_REQUESTED** [review]
commit message not appropriate

**Review by carina29 — APPROVED** [review]

File: sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java [hunk 1]
```diff
@@ -170,27 +170,32 @@ private void saveContactsFromLineListing(LineListingLayout lineListingForm, Link
 	}
 
 	public void openLineListingWindow(EventDto eventDto, Set<EventParticipantIndexDto> eventParticipantIndexDtos) {
-		if (eventParticipantIndexDtos == null || eventParticipantIndexDtos.isEmpty()) {
-			return;
-		}
+		if (eventParticipantIndexDtos.size() == 0) {
```
Comment (StefanKock) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java:173]: This is not null-safe anymore. Let's hope no one calls this with `null`

File: sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java [hunk 2]
```diff
@@ -170,27 +170,32 @@ private void saveContactsFromLineListing(LineListingLayout lineListingForm, Link
...
+			new Notification(
```
Comment (StefanKock) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java:174]: The whole message block could have been added to the pre-existing if clause without adding an else statement.

**Review by StefanKock — COMMENTED** [review]
Added some remarks but my initial request was fixed

**Review by StefanKock — APPROVED** [review]

**Pull request merged**